indico/modules/events/editing/client/js/components/FileTypeManager.jsx

```diff
@@ -41,6 +42,18 @@ function fileTypesReducer(state, action) {
   }
 }
 
+function statusIcon(fileParam, iconName, text) {
```

Comment:
It's a component so it should be named `StatusIcon`, and used as `<StatusIcon ... />` below instead of `{statusIcon(...)}`